server/service-manager.js

```diff
@@ -0,0 +1,49 @@
+var debug = require('debug')('MeshModels.server.ServiceManager');
+
+function ServiceManager() {
+  debug('ServiceManager created');
+}
+
+function onServiceUpdate(service, callback) {
+  debug('onServiceUpdate(%j)', service);
+  setImmediate(callback);
+}
+ServiceManager.prototype.onServiceUpdate = onServiceUpdate;
+
+function onServiceDestroy(service, callback) {
+  debug('onServiceDestroy(%j)', service);
+  setImmediate(callback);
+}
+ServiceManager.prototype.onServiceDestroy = onServiceDestroy;
```

Comment:
Should Destroy be Delete, so it parallels the rest API more closely, or is it better to use a different name to abstract away?